utils/xlsx_reader.py

```diff
@@ -37,7 +38,14 @@ def create_library_item(session, model, **kwargs):
     if library_item:
         return library_item
     else:
-        instance = model(**kwargs)
+        language = choice(['polish', 'other', 'english'])
+        if model.__name__ == "Book":
+            rand_date = datetime.strptime(str(randint(1978, 2018)), '%Y')
```

Comment:
'Magic numbers' for the rand_date - in the future one has to update the newest year.
Better to get the current year as a datetime object and cast it to int for random generator.